tests/core/test_core.py

```diff
@@ -272,18 +272,23 @@ def test_narrow_to_all_mentions(
         msg_ids = {widget.original_widget.message['id'] for widget in widgets}
         assert msg_ids == id_list
 
-    def test_view_in_browser(self, mocker, controller):
+    @pytest.mark.parametrize('server_url', [
```

Comment:
Re the commit title, this doesn't always open a message in _topic_ narrow (e.g. it enables PMs/huddles as well).